scripts/update_package_json_files.js

```diff
@@ -13,14 +13,18 @@ function bumpVersionInVSCodeRepo({ version, name, displayName, description, prev
   writeJsonToPackageJson({ content: content, path: vscodePackageJsonPath })
 }
 
-function bumpLSVersionInExtension({ version }) {
+async function bumpLSVersionInExtension({ version }) {
   const vscodePackageJsonPath = path.join(__dirname, '../packages/vscode/package.json')
   let content = getPackageJsonContent({ path: vscodePackageJsonPath })
   content['dependencies']['@prisma/language-server'] = version
   writeJsonToPackageJson({ content: content, path: vscodePackageJsonPath })
+  
+  // Update pnpm-lock.yaml after package.json changes
+  console.log('Running pnpm install to update pnpm-lock.yaml...')
+  await execa('pnpm', ['i'], { cwd: path.join(__dirname, '..'), stdio: 'inherit' })
```

Comment:
nit: I think shorthand commands should be reserved for interactive shell usage; scripts should be using full names of subcommands, full `--something` option names instead of shorthand `-s`, etc